GiftedListView.js

```diff
@@ -13,16 +13,12 @@ var {
 
 
 // small helper function which merged two objects into one
-function MergeRecursive(obj1, obj2) {
-  for (var p in obj2) {
-    try {
-      if ( obj2[p].constructor==Object ) {
-        obj1[p] = MergeRecursive(obj1[p], obj2[p]);
-      } else {
-        obj1[p] = obj2[p];
-      }
-    } catch(e) {
-      obj1[p] = obj2[p];
+function MergeRowsWithHeaders(obj1, obj2) {
+  for(var p in obj2){
+    if(obj1[p] instanceof Array && obj1[p] instanceof Array){
```

Comment:
this line should be:
`if(obj1[p] instanceof Array && obj2[p] instanceof Array){`